pkg/artifacts.go

```diff
@@ -0,0 +1,33 @@
+package client
```

Comment:
? Why client  instead of pkg?

Reply:
I think `pkg` is not a good name for package and has not relationship with Jenkins client.

Reply:
Please see https://github.com/minio/minio-go

Reply:
IMO, `client` is not a good name. All source files of this project belong to a client. How about creating some sub-directory. Such as:

```
pkg/job
...
pkg/artifact
...
pkg/credential
```

See also the [Golang package best practices](https://rakyll.org/style-packages/).